### DISCUSSION TITLE
M5Stack ATOM lite

### DISCUSSION THREAD

**Discussion by KalaNagHTD** [raomin/ESPAltherma#237] [General]
Hello all!

I would like to exchange my M5StickC with an M5Stack ATOM lite.

Can the experts here tell me what to change in the ESPAltherma code? 
Is it sufficient to just adjust the GPIOs and set the Environment of the IDE to m5stack-atom?
I'm sure there is more to do...

Thanks in advance!

Regards Karl

**Comment by KalaNagHTD** [comment]
The tiny Atom Lite works!
I just had to change the I/O ports from GPIO 36/26 to GPIO 21/25 in the setup.h.
All the rest including the environment for the M5Stick-C remained unchanged.
My onboard 5V supply can feed the M5Stack Atom Lite which was not possible with the M5Stick-C.

![0ACE47BB-4222-4AEF-A93E-431693073E4C](https://user-images.githubusercontent.com/77481230/216618729-9b4dc9cb-8f65-40fd-bc78-76472dd21ae6.jpeg)
![EC1637CD-EC96-46D3-BF4D-3A33DBED5002](https://user-images.githubusercontent.com/77481230/216618719-2d2abf77-b0aa-439b-8ffd-5be81846a398.jpeg)

**Reply by KalaNagHTD** [reply]
Even better is the use of the Grove connector.
I had to change the I/O ports from GPIO 36/26 (M5Stick-C) to GPIO 32/26 in the setup.h.

![IMG_5836](https://user-images.githubusercontent.com/77481230/217776317-7c08010e-67cb-4ea0-9f58-97cf2ae207f8.jpg)
![IMG_5837](https://user-images.githubusercontent.com/77481230/217776325-5b88ba03-b024-44eb-a908-b7268ee972b3.jpg)

**Comment by danisowa** [comment]
Hello i plan to give ESPAltherma and Atom Lite a try.
Did you use a 5v / 3.3V Level Shifter bewteen X10A and the Atom Lite?

**Reply by KalaNagHTD** [reply]
No! 
The Atom has been working perfectly fine for 2 years now.

**Comment by bazcurtis** [comment]
I am very new to this and want to make sure I am buying the right parts. Is this the correct device? [Atom Lite](https://uk.robotshop.com/products/m5stack-atom-lite-esp32-development-kit)

I tried to look up the Gove connector, but couldn’t find anything. Where is the best place to get that cable?

**Reply by KalaNagHTD** [reply]
Search for Grove, not Gove. It is a Arduino standard connector.
